### PR TITLE
Fix spelling

### PR DESCRIPTION
**Pull request by hugovk**
| Q             | A
| ------------- | ---
| Bug fix?      | yes 
| New feature?  | no 
| BC breaks?    | no
| Deprecations? | no
| License       | MIT

### CODE REVIEW

File: data/schema.rdfa [hunk 1]
```diff
@@ -3183,7 +3183,7 @@ The place is __open__ if the [[opens]] property is specified, and __closed__ oth
```
Comment (dunglas) [inline data/schema.rdfa]: This file comes from Schema.org, those fix should be contributed upstream: https://github.com/schemaorg/schemaorg

File: data/v1.owl [hunk 2]
```diff
@@ -528,7 +528,7 @@ Note 3: If the shop re-opens on the same day of the week or set of days of the w
     </rdfs:domain>
```
Comment (dunglas) [inline data/v1.owl]: This file comes from Good Relations, these changes should be contributed upstream: https://github.com/mfhepp/goodrelations

**Comment by dunglas** [comment]
Thanks @hugovk

File: data/schema.rdfa (hunk 1 above)
Comment (hugovk) [reply data/schema.rdfa]: Thanks, sent upstream: https://github.com/schemaorg/schemaorg/pull/2390.

File: data/v1.owl (hunk 2 above)
Comment (hugovk) [reply data/v1.owl]: They're in the process of migrating code to there from Bitbucket, I'll check back in a bit and contribute it back later. Thanks!

**Pull request merged**